flink-streaming-java/src/main/java/org/apache/flink/streaming/runtime/io/CheckpointBarrierAligner.java

```diff
@@ -318,6 +325,13 @@ public long getAlignmentDurationNanos() {
 		}
 	}
 
+	private void unblockCheckpoint(Integer channelToUnblock) {
```

Comment:
nit: Integer -> int

Reply:
I suggest renaming `unblockCheckpoint` to `resumeConsumption` to unify all the stacks.